account_invoice_triple_discount/models/line_triple_discount.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017 Tecnativa - David Vidal
+# Copyright 2017 Tecnativa - Pedro M. Baeza
+# Copyright 2021 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class LineTripleDiscount (models.AbstractModel):
+    _name = "line.triple_discount"
```

Comment:
could you rename to ``line.triple.discount.mixin`` to mention explicitely that it is an abstract model ?

Reply:
Thanks for the review! Done

Reply:
between triple_discount and triple.discount : not a blocking point. matter of taste ;-)